ci-operator/step-registry/ipi/conf/azure/oidc-creds-provision/ipi-conf-azure-oidc-creds-provision-commands.sh

```diff
@@ -102,11 +102,6 @@ popd
 echo "CR manifest files:"
 ls "/tmp/credrequests"
 
-# create metadata so cluster resource group is deleted in ipi-deprovision-deprovision
-cat > ${SHARED_DIR}/metadata.json << EOF
-{"infraID":"${CLUSTER_NAME}","azure":{"region":"${REGION}","resourceGroupName":"${CLUSTER_NAME}"}}
-EOF
```

Comment:
@jstuever I am deleting the lines, because once `ipi-conf-azure-provisioned-resourcegroup` is applied in the install workflow, installer can generate the metadata.json correctly.